runtime/permissions.rs

```diff
@@ -2086,12 +2092,23 @@ fn permission_prompt(message: &str, name: &str) -> bool {
       Some(v) => v,
     };
     match ch.to_ascii_lowercase() {
-      'y' => return true,
-      'n' => return false,
+      'y' => {
+        eprint!("\x1B[4A\x1B[0J");
```

Comment:
Maybe some explanation of what this is would be helpful. Also, looks like it could be extracted out to the line above the match.

Reply:
Done